### PR TITLE
feat: update rack attack ip limit

### PR DESCRIPTION
**Pull request by scmmishra**
This PR updates rack attach per-ip limit. Also, allows fetching the value from the ENV

### CODE REVIEW

**Comment by scmmishra** [comment]
@pranavrajs @sojan-official should I implement a redis based safelist too?

File: config/initializers/rack_attack.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@ def path_without_extentions
   #
   # Key: "rack::attack:#{Time.now.to_i/:period}:req/ip:#{req.ip}"
 
-  throttle('req/ip', limit: 300, period: 1.minute, &:ip)
+  throttle('req/ip', limit: ENV.fetch('RACK_ATTACK_LIMIT', 300), period: 1.minute, &:ip)
```
Comment (pranavrajs) [inline config/initializers/rack_attack.rb:49]: Let us use 3000 here

**Review by pranavrajs — CHANGES_REQUESTED** [review]
One minor comment

**Comment by pranavrajs** [comment]
> @pranavrajs @sojan-official should I implement a redis based safelist too?

Let us start with this. If it becomes a problem, let us implement it.

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**